just-install.json

```diff
@@ -313,6 +313,34 @@
       },
       "version": "3.15.1"
     },
+    "codeblocks": {
+      "installer": {
+        "kind": "custom",
```

Comment:
This is a NSIS installer, so you can change it to `nsis` and remove the `options`.

Reply:
@geek1011 I changed it to `nsis`.